### PR TITLE
watchlist (redux) state added

### PR DESCRIPTION
**Pull request by smarquezmgh**

### CODE REVIEW

File: src/ui/common/externalAssetsLink.ui.tsx [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
-        {land.metaverse === "Decentraland" && <Image src='/images/decentraland-mana-logo.png' width={55} height={55} alt='decentraland' className='rounded-full p-2 bg-nm-highlight absolute bottom-3 left-3' />}
-        {land.metaverse === "Somnium Space" && <Image src='/images/somnium-space-logo.png' width={55} height={55} alt='somniun-space' className='rounded-full p-2 bg-nm-highlight absolute bottom-3 left-3' />}
+        {metaverse === "sandbox" && <Image src='/images/the-sandbox-sand-logo.png' width={55} height={55} alt='sandabox' className='rounded-full p-2 bg-nm-highlight absolute bottom-3 left-3' />}
+        {metaverse === "decentraland" && <Image src='/images/decentraland-mana-logo.png' width={55} height={55} alt='decentraland' className='rounded-full p-2 bg-nm-highlight absolute bottom-3 left-3' />}
+        {metaverse === "somnium-space" && <Image src='/images/somnium-space-logo.png' width={55} height={55} alt='somniun-space' className='rounded-full p-2 bg-nm-highlight absolute bottom-3 left-3' />}
```
Comment (nanSer0) [inline src/ui/common/externalAssetsLink.ui.tsx]: use enum for that

File: src/ui/common/landsMenu.ui.tsx [hunk 2]
```diff
@@ -1,55 +1,58 @@
...
 export default function LandsMenuUI({ metaverse, setMetaverse, form, isBorder }: LandsMenuUIProps) {
-
   return (
     <div className={`w-full flex items-center justify-center  py-16 ${isBorder ? "border-b border-nm-remark" : ""}`}>
       {
         form == ButtonForm.Horizontal ?
           <>
-            {(Object.keys(Metaverses) as Array<keyof typeof Metaverses>).map((key) => (
-              <button
+            {(Object.keys(MetaverseOptions) as Array<MetaverseOptionsKey>).map((key) => {
```
Comment (nanSer0) [inline src/ui/common/landsMenu.ui.tsx]: we have a util for Object.keys

File: src/ui/common/landsMenu.ui.tsx [hunk 3]
```diff
@@ -1,55 +1,58 @@
...
-            ))}
+
+            }
+            )}
           </>
           :
           <>
-            {(Object.keys(Metaverses) as Array<keyof typeof Metaverses>).map((key) => {
-              if (Metaverses[key] === Metaverses.ALL) {
+            {(Object.keys(MetaverseOptions) as Array<MetaverseOptionsKey>).map((key) => {
```
Comment (nanSer0) [inline src/ui/common/landsMenu.ui.tsx]: we have a util for Object.keys

File: src/ui/heatmap/hotDeals/hotDealsCard.ui.tsx [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
 
   const metaverseImage = (getMetaverseImg());
 
   function getMetaverseImg() {
-    if (metaverseSelected == Metaverses.DECENTRALAND) return '/images/decentraland-mana-logo.png';
-    if (metaverseSelected == Metaverses.SANDBOX) return '/images/the-sandbox-sand-logo.png';
-    if (metaverseSelected == Metaverses.SOMNIUM) return '/images/somnium-space-cube-logo.webp';
+    if (metaverseSelected == "decentraland") return '/images/decentraland-mana-logo.png';
+    if (metaverseSelected == "sandbox") return '/images/the-sandbox-sand-logo.png';
+    if (metaverseSelected == "somnium-space") return '/images/somnium-space-cube-logo.webp';
```
Comment (nanSer0) [inline src/ui/heatmap/hotDeals/hotDealsCard.ui.tsx]: use enum for that

**Review by nanSer0 — CHANGES_REQUESTED** [review]

**Review by nanSer0 — APPROVED** [review]

**Comment by OswaldUnity** [comment]
Please fix the linter errors @smarquezmgh @nanSer0

**Pull request merged**